loop/management/commands/maharashtra_add_agg_mandi.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+from loop.models import LoopUser, Mandi, LoopUserAssignedMandi
+
+import pandas as pd
+
+class Command(BaseCommand):
+
+	def handle(self, *args, **options):
+		df = pd.DataFrame()
+
+		df = pd.read_excel('Maharashtra Admin Data Sheet.xlsx', sheetname='Aggregator - Markets')
+		print df
+		status = [None]*len(df['Name (Regional)'])
+		df['Status'] = pd.Series(status).values
+		df['Exception'] = pd.Series(status).values
+
+		for i, row in df.iterrows():
+			try:
+					aggregator_name = row['Name (Regional)'].strip()
+					mandi_name = row['Market (Regional)'].strip()
+					aggregator = LoopUser.objects.get(name=aggregator_name)
+					mandi = Mandi.objects.get(mandi_name = mandi_name)
+					aggregator_mandi = LoopUserAssignedMandi(loop_user = aggregator, mandi = mandi)
```

Comment:
First check weather this relationship already in table, otherwise it will create duplicate entry.